docs/examples/general/data_loading/webdataset.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.            initial_fill = dali.pipeline.Pipeline.current().max_batch_size
</pre><p>I find this too little to do shuffling. Maybe use the default in DALI readers (1000)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>alright I just wanted it to be defined as something related to the batch_size, maybe 16xbatch_size would be alright? (the datasets themselves are only 3k samples)</p>

Reply:
I think the batch size is irrelevant here. I'd choose something like 512, or so